### DISCUSSION TITLE
delete button hasmany resource

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#2052] [Help]
how can i hide the edit and delete buttons of a hasmany relationship resource?

**Comment by danharrin** [comment]
Using a policy for the `update` and `delete` methods?

**Reply by jcgmalave** [reply]
No, how could I implement it?

**Reply by danharrin** [reply]
You'd need a policy for your related model

https://laravel.com/docs/9.x/authorization#creating-policies
